lib/public/SystemTag/TagAssignedEvent.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2016-2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+namespace OCP\SystemTag;
+
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IWebhookCompatibleEvent;
+
+/**
+ * Event class for when a system tag is assigned to an object
+ *
+ * @since 32.0.0
+ */
+class TagAssignedEvent extends Event implements IWebhookCompatibleEvent {
+	protected string $objectType;
+	protected string $objectId;
+	/** @var int[] */
```

Comment:
```suggestion
	/** @var list<int> */
```
Please type it like this everywhere.